lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationElementExtension.cs

```diff
@@ -251,12 +257,10 @@ public void WriteTo(XmlWriter writer)
             {
                 if (_xmlSerializer != null)
                 {
-                    //Fx.Assert((this.dataContractSerializer == null && this.outerName == null && this.outerNamespace == null), "Xml serializer cannot have outer name, ns");
                     _xmlSerializer.Serialize(writer, _extensionData);
```

Comment:
For the rest of the Fx.Assert's that have been removed, put them back but use Debug.Assert